### ISSUE TITLE
The adjust rider size option rounds to 0.5 increments

### ISSUE THREAD

**Issue by ddbrown30** [Saibot393/Rideable#74]
The current way that adjust rider size works doesn't line up with the option. It seems Foundry has a forced restriction that token dimensions can only be changed in 0.5 increments. It simply doesn't give the fine control that the option implies. This causes weird behaviour with the option depending on what value you've set. For example, the default value of 0.65 means a 1x1 token will shrink to be a 0.5x0.5 token whereas a 3x3 token will shrink to 2x2. What's worse, though, is that a value of 0.7 on a 1x1 doesn't scale it at all.

This is something that likely can't be fixed directly as it's a restriction in Foundry. That said, if you rework the logic (or just add another option so it doesn't break it for existing users) to operate on the scale value rather than on the dimensions, you could have a much more precise control on the adjustment.

**Comment by Saibot393** [comment]
Correct, there is nothing i can do to make the dimension size scaling work with smaller steps. I will look into adding an option for the graphical scaling.
